src/detours.cpp

```diff
@@ -18,6 +18,8 @@
 
 #define NOTHROW
 
+#define ERROR_DYNAMIC_CODE_BLOCKED       1655L // see winerror.h, as this error isn't found in some SDKs
```

Comment:
Consider putting this in detours.h under the DETOURS_INTERNAL ifdef, incase it needs to be used elsewhere in the future.